### PR TITLE
Allows for Scroll to close Meter Toast iframe

### PR DESCRIPTION
**Pull request by elijahsoria**
Implements the ability to close the meter toast iframe by any mousedown, touchstart, click, and wheel events. Makes the window's body hidden while the iframe is visible to avoid scrolling during iframe touch events. Adds tests.

### CODE REVIEW

File: src/runtime/meter-toast-api.js [hunk 1]
```diff
@@ -68,12 +68,25 @@ export class MeterToastApi {
...
+      .then((_unusedDialog) => {
+        // Allow closing of the iframe with any scroll or click event.
+        this.win_.addEventListener('click', this.sendCloseRequestFunction_);
+        this.win_.addEventListener(
+          'touchstart',
+          this.sendCloseRequestFunction_
+        );
+        this.win_.addEventListener('mousedown', this.sendCloseRequestFunction_);
+        this.win_.addEventListener('wheel', this.sendCloseRequestFunction_);
+        // Making body 'hidden' to prevent scrolling while swiping on the iframe.
```
Comment (ChrisAntaki) [inline src/runtime/meter-toast-api.js:86]: Could you update the comment to specify the body's "overflow" property is being set to "hidden", not the body itself?

File: src/runtime/meter-toast-api.js [hunk 2]
```diff
@@ -85,10 +98,15 @@ export class MeterToastApi {
   }
 
   /**
-   * Removes the event listener that closes the iframe.
+   * Removes the event listeners that close the iframe and makes the body visible.
```
Comment (ChrisAntaki) [inline src/runtime/meter-toast-api.js:101]: "makes" => "make"

**Review by ChrisAntaki — APPROVED** [review]

File: src/runtime/meter-toast-api.js (hunk 1 above)
Comment (elijahsoria) [reply src/runtime/meter-toast-api.js]: Done

File: src/runtime/meter-toast-api.js (hunk 2 above)
Comment (elijahsoria) [reply src/runtime/meter-toast-api.js]: Done

**Review by ChrisAntaki — APPROVED** [review]

**Review by ChrisAntaki — APPROVED** [review]

**Pull request merged**